src/plugin/code.js

```diff
@@ -1,5 +1,85 @@
 figma.showUI(__html__, { width: 400, height: 540 });
 
+const differenceRectangleIdList = [];
+
+const CONSTANTS = {
+  MODIFIED_FILLS: {
+    type: "SOLID",
+    color: { r: 0.435, g: 0.831, b: 0.505 },
+  },
+  NEW_FILLS: {
+    type: "SOLID",
+    color: { r: 0.435, g: 0.831, b: 0.505 },
+  },
+  RECT_OPACITY: 0.2,
+};
+
+const isOwnProperty = (targetObject, targetProperty) => {
+  return Object.prototype.hasOwnProperty.call(targetObject, targetProperty);
+};
+
+const renderDifferenceRectangle = (differences, modifiedFrames) => {
+  for (const nodeId in differences) {
+    if (isOwnProperty(differences, nodeId)) {
+      const modifiedNode = differences[nodeId];
+      const { type, differenceInformation, frameId } = modifiedNode;
+      const { x, y, width, height } = modifiedNode.position;
+
+      const differenceRectangle = figma.createRectangle();
+
+      differenceRectangle.resize(width, height);
+      differenceRectangle.x = x;
+      differenceRectangle.y = y;
+      differenceRectangle.opacity = CONSTANTS.RECT_OPACITY;
+
+      switch (type) {
+        case "NEW":
+          differenceRectangle.fills = [CONSTANTS.NEW_FILLS];
+          differenceRectangle.setPluginData(
+            "differenceInformation",
+            "선택하신 이전 버전에는 존재하지 않는 노드 입니다!",
```

Comment:
P2: 해당 부분만 띄어쓰기 붙여주시면 좋을 것 같습니다 :)
```suggestion
            "선택하신 이전 버전에는 존재하지 않는 노드입니다!",
```

Reply:
수정하겠습니다..! 🥹